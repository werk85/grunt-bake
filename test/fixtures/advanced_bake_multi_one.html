<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<!--(bake includes/include-one.html)-->
	</body>
</html>
