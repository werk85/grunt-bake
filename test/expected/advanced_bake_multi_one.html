<!DOCTYPE html>
<html>
	<head></head>
	<body>
		Include One
	</body>
</html>
